.circleci/config.yml

```diff
@@ -158,6 +158,25 @@ jobs:
       - store_artifacts: *solc_artifact
       - persist_to_workspace: *all_artifacts
 
+  test_check_spelling:
+    docker:
+      - image: buildpack-deps:artful
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: build
+      - run:
+          name: Install dependencies
+          command: |
+            apt-get -qq update
+            apt-get -qy install python-pip
```

Comment:
Does it makes sense to use `- image: circleci/python:[TAG]` here which might already include `pip`?